src/pages/who-we-are/_OurOffices.tsx

```diff
@@ -126,7 +126,14 @@ const OurOffices = () => {
 
             <NumberSection columns="1fr 1fr 1fr 1fr" column_gap="120px" row_gap="4rem">
                 {our_offices_count.map(({ count, title }) => (
-                    <StyledFlex fd="column" key={title}>
+                    <StyledFlex
+                        fd="column"
+                        key={
+                            typeof title === 'string'
+                                ? title
+                                : title.props.translate_text
```

Comment:
```suggestion
                            typeof title === 'string'
                                ? title
                                : title.props.translate_text
```
Same here?

Reply:
Hi @jim-deriv , you are right, I have redone it as you said, but on [deriv.com](http://deriv.com/) we use buildTime Api and don’t need to use an optional.